model/ResultsService.php

```diff
@@ -633,7 +633,7 @@ public function structureItemVariables($itemVariables, $filter)
                 } else {
                     $variableDescription['isCorrect'] = 'unscored';
                 }
-
+                $variableDescription['isExternallyGraded'] = (bool)$itemVariable->isExternallyGraded ?? false;
```

Comment:
I notice in other [PR](https://github.com/oat-sa/extension-tao-outcomerds/pull/151/files) you put the logic inside parenthesis. Do you think is needed here? At least is more clear.